components/centraldashboard/public/assets/i18n/languages.json

```diff
@@ -65,6 +65,17 @@
             "landingPage.slackLink": "https://statcan-aaw.slack.com",
             "landingPage.firstTime": "This seems to be the first time you access Kubeflow, we are happy to see you!",
             "landingPage.autoNamespace": "Your namespace is automatically generated.",
+            "notebookDefaultCard.connect": "Connect",
+            "notebookDefaultCard.create": "Create",
+            "notebookDefaultCard.details": "Details",
+            "notebookDefaultCard.name": "Name",
+            "notebookDefaultCard.type": "Type",
+            "notebookDefaultCard.helpText": "The default notebook will automatically start upon login into Kubeflow",
+            "notebookDefaultCard.txtDefaultNotebook": "Default Notebook",
+            "notebookDefaultCard.txtDataLoading": "The data is loading",
+            "notebookDefaultCard.txtNoDefaultNotebook": "If you see this note, the system has not detected a default notebook for this namespace. You may click on 'Create' below to generate one.",
```

Comment:
I dont think we need to tell the users that the system didnt find a default notebook. If there is no default notebook, I think we should just let them know that they can create one.

Reply:
I like the idea of more info to the user. Like, after maybe they want to create another one, but they remember that it exists and therefore cant?

Reply:
Its not really more info to the users. Its like, we dont say "If you see this message, the system has not detected notebooks" if someone doesn't have notebooks in their namespace, we just say "no notebooks". 
I feel like a better way to phrase this is just "No default notebook found". And if there is no notebook found, then it's obvious that the "create" button can be used to create one.

Reply:
Also reguardless of if you change that message or not, the french message doesnt match, still talks about automatic start.